src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,58 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String SUPPLY = "supply";
+    public static final String BUY = "buy";
+    public static final String RESULT = "result";
+    public static final String DIVIDER = ",";
+    public static final String C_R_L_F = System.lineSeparator();
+    public static final int POS_OF_AMOUNT = 1;
+    public static final int POS_OF_NAME = 0;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String input = getFromFile(fromFileName);
+        writeToFile(input, toFileName);
+    }
+
+    public void writeToFile(String data, String toFileName) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName))) {
+            bufferedWriter.write(data);
+            bufferedWriter.flush();
+        } catch (IOException e) {
+            throw new RuntimeException("Cant write data to file", e);
+        }
+    }
 
+    private String getFromFile(String fromFileName) {
+        int suppliesSummary = 0;
```

Comment:
In this method you read data from file and make a result. It's better to divide this logic to separate methods

Reply:
Not resolved, it is possible to read the file first, and then provide fileContent to method responsible for report creation and make calculation logic